### ISSUE TITLE
Issues wrapping lines with ' and "

### ISSUE THREAD

**Issue by lordmauve** [kneufeld/consolemd#12]
Passing  `-w` causes lines with single and double quote characters to be wrapped incorrectly - both with additional spaces, and not fitting to the expected line width.

Using consolemd 0.5.0, I ran `consolemd -w 40` with these two input lines:

```
I don't know what kind of monster we've made here, but it's definitely
of the "three arms, no legs" variety.
```

The output was

```
I don ' t know what kind of monster we ' ve made here, but it ' s definitely
of the " three arms, no legs "  variety.
```

... the first line of which is 76 characters long. The spaces around quotes are also unexpected.

**Comment by kneufeld** [comment]
Thank you for using `consolemd` and for reporting this bug.

Unfortunately I'm not sure if this is fixable... 

If you run `consolemd -d -w40` with that input you'll see that at each quote character a new `text` tag is generated. Each of the `text` tags is wrapped individually so in this case no text block is longer than 40 chars so wrapping won't actually occur. These tags come from `commonmark` and I'm definitely not opening _that_ can of worms.

Hmm... maybe the raw text needs to be wrapped and then outputted with `--no-soft-wrap`. I'll give it some thought and testing.

**Comment by kneufeld** [comment]
So good news is that per-processing the text works but the simple solution merges all paragraphs into a single text blob. So each paragraph needs to be individually processed but getting that text out of commonmark is... not the best.

The good news is that extra spaces will get fixed either way.

**Comment by kneufeld** [comment]
Fixed and pushed version 0.5.1 to pypi
